pkg/k8s/openebs.go

```diff
@@ -117,3 +118,19 @@ func CreateCStorPoolClusters(captenConfig config.CaptenConfig) error {
 	_, err = poolClusterClient.Update(context.TODO(), poolCluster, metav1.UpdateOptions{})
 	return err
 }
+
+func CreateCStorPoolClusterWithRetries(captenConfig config.CaptenConfig) error {
+	retries := 3
+	for i := 0; i < retries; i++ {
+		err := CreateCStorPoolClusters(captenConfig)
```

Comment:
why cstor pool create faling at first attempt ?
actual error not getting logged.

use retry pattern function for retries